api-specs/readme.md

```diff
@@ -1,13 +1,15 @@
-This is a temporary directory that exists solely for the purpose of generating "Run on Insomnia" buttons. 
+This directory stores API specs for the creation of the  "Run on Insomnia" buttons found on the [API docs page](https://docs.konghq.com/api/). This directory is also the source of truth for the beta Kong Gateway Enterprise and Open Source Admin API specs. 
```

Comment:
I'd put the platform-api info up here too, so that "source of truth" info is in the same place.